content/docs/user-guide/external-dependencies.md

```diff
@@ -72,12 +54,12 @@ $ dvc run -n download_file
 
 ```dvc
 $ dvc run -n download_file
-          -d azure://my-container-name/data.txt \
```

Comment:
@jorgeorpinel do we really need updates like this? can we at least isolate and do them in a separate PR - so that it's clear - nothing to review, just merge

Reply:
Yes, I can split it. This PR is a draft. I try to check them myself before requesting a review.

Please keep in mind that I put all the misc. updates into this `jorge` branch, and they mount up quickly, so it's not easy to balance the size of these generic PRs vs. the number of specialized branches and PRs.

Reply:
So going back to this. This PR's main thing is "Remote URL revisions in multiple docs" as mentioned in the description. This includes consistency in these sample hosts and paths, that only uses `-` in a minority of the cases, so I changed them to no `-`. Again, that's what this PR is mainly about (see other similar changes in the remote add ref).

I will try to extract other things not related to this idea before I request a review.

Thanks